fusion-cli/test/e2e/split-translations/test.js

```diff
@@ -29,6 +29,18 @@ test('`fusion build` app with split translations integration', async () => {
     content.includes('__MAIN_TRANSLATED__'),
     'app content contains translated main chunk'
   );
+  t.ok(
```

Comment:
Purpose of tests in this file:
- i18n manifest (chunk-translation-map) properly includes dynamic translations arrays
- node plugin uses static analysis output to locate matching translations